api_tests/nodes/views/test_node_relationship_institution.py

```diff
@@ -0,0 +1,159 @@
+from nose.tools import *
+
+from framework.auth.core import Auth
+
+from tests.base import ApiTestCase
+from tests.factories import InstitutionFactory, AuthUserFactory, NodeFactory
+
+from api.base.settings.defaults import API_BASE
+
+class TestNodeRelationshipInstitution(ApiTestCase):
```

Comment:
I would add tests for checking if your request format is incorrect: if type is missing, and your just have id, or vice versa.